Fruitties/shared/src/commonMain/kotlin/com/example/fruitties/database/AppDatabase.kt

```diff
@@ -29,6 +29,7 @@ abstract class AppDatabase : RoomDatabase() {
 
 // The Room compiler generates the `actual` implementations.
 @Suppress("NO_ACTUAL_FOR_EXPECT")
-expect object AppDatabaseConstructor : RoomDatabaseConstructor<AppDatabase>
-
+expect object AppDatabaseConstructor : RoomDatabaseConstructor<AppDatabase> {
+    override fun initialize(): AppDatabase
```

Comment:
Is the override and function declaration necessary? I think this can just be the expect declaration.

`expect object AppDatabaseConstructor : RoomDatabaseConstructor<AppDatabase>`

Reply:
I got a compile error when compiling without the override, which is why I added it.

```
Object 'AppDatabaseConstructor' is not abstract and does not implement abstract member 'initialize'.
```

Reply:
We confirmed that the error you see happens during some Gradle builds. It's not required when building for and Android or iOS target, but it is required for building common-only source, which happens during Android Studio -> Build -> Make Project or commands like ./gradlew build. Let's include the empty override as you wrote it to avoid the distracting errors.